bowling/bowlingLane-ut.cpp

```diff
@@ -1,26 +1,27 @@
 #include <gtest/gtest.h>
+#include <memory>
 #include <string>
 #include <vector>
 
 #include "bowlingLane.hpp"
 
 struct TestBowlingLine : public ::testing::Test {
-    BowlingLane bl = BowlingLane();
-    std::string results{};
-    std::vector<std::pair<int, int>> points{{1, 1}, {1, 2}};
+    std::vector<std::unique_ptr<Player>> players;
+    std::vector<std::pair<int, int>> points{{1, 2}, {1, 2}, {1, 2}, {1, 2}, {1, 2}, {1, 2}, {1, 2}, {1, 2}, {1, 2}, {1, 2}}; 
 };
 
-TEST_F(TestBowlingLine, checkIfSetPlayerAddPlayer) {
-    bl.setPlayer("player", points);
+TEST_F(TestBowlingLine, checkPrintResultToScreen) {
+    players.emplace_back(std::make_unique<Player>("player", points));
+    BowlingLane bl(std::move(players));
+    EXPECT_TRUE(bl.checkGameStatus());
+    
     bl.checkGameStatus();

```

Comment:
bl.checkGameStatus() is used twice. Once under assertion and the other one without assertion.